src/main/java/org/isf/sms/manager/SmsManager.java

```diff
@@ -102,6 +102,7 @@ else if (textLenght > MAX_LENGHT && split) {
 			for (String part : parts) {
 				Sms sms = new Sms();
 				sms.setSmsNumber(number);
+				sms.setSmsDate(smsToSend.getSmsDate());
```

Comment:
The date has to be set because the object marks the field as NON-NULL in the column definition.   This is one of the cases that I mentioned that the newer spring boot and its dependencies are more strict.

Reply:
I see... actually the field has a default value `SMS_DATE TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP`, it would be nice to ask Spring authors how to leverage SQL defaults otherwise are they useless?

Reply:
It might not be Spring; it may be Hibernate.  I am just not sure.

Reply:
good point. what about using annotations like `@Column(nullable = false, columnDefinition = "TIMESTAMP DEFAULT CURRENT_TIMESTAMP")` ?

Reply:
I have found another solution.   The  "problem" is that we have an existing database; those annotations are used when generating or regenerating tables.   The suggested change isn't pretty but it is the best one I could find on the web that works with our setup.

Reply:
I remember this kind of solution, it's ok for me, tested?

Reply:
yes; the tests all pass